docs/contributing.rst

```diff
@@ -46,6 +46,19 @@ a human. All code style enforcement must be automated to avoid bikeshedding
 and losing time.
 
 
+Logging
+=======
+
+The ddtrace logger should be used log events in the dd-trace-py library. Use `ddtrace.internal.logger.get_logger(__name__)` to initialize/retrieve an instance of the ddtrace logger (DDLogger).
+
+To ensure the ddtrace library produces consistent and secure logs the following best practices should be followed:
```

Comment:
I believe you need to add an extra linebreak before the bullet list to get it to format properly.